TLM/TLM/U/UIScaler.cs

```diff
@@ -49,14 +49,14 @@ public static class UIScaler {
         public static float MaxWidth {
             get {
                 float ret = Config.GuiScaleToResolution ? BaseResolutionX : Screen.width;
-                return ret / Config.GuiScale;
+                return ret / (Config.GuiScale * 0.01f);
```

Comment:
no magic numbers plz. make the 0.01f a variable.

Reply:
@kianzarrin I don't remember what range GuiScale is (probably 30...200+) but if it's percentage then it does not look right to me. For 100 it will return `ret/1f` which is ok,  but for 200 it will be `ret/2f`, so 2x smaller?? 🤔